tests/test_indexer.py

```diff
@@ -194,29 +195,73 @@ def test_indexed_file_access_error(self):
                                                          files=files,
                                                          excluded_files=[inaccesssible_filename.replace(".", "_")])
 
+    def test_notify(self):
+        def _notify(subscription, bundle_id="i.v"):
+            notify(subscription_id=subscription["id"], subscription=subscription, bundle_id=bundle_id, logger=logger)
+        with self.assertRaisesRegex(requests.exceptions.InvalidURL, "Invalid URL 'http://': No host supplied"):
+            _notify(subscription=dict(id="", es_query={}, callback_url="http://"))
+        with self.assertRaisesRegex(AssertionError, "Unexpected scheme for callback URL"):
+            _notify(subscription=dict(id="", es_query={}, callback_url=""))
+        with self.assertRaisesRegex(AssertionError, "Unexpected scheme for callback URL"):
+            _notify(subscription=dict(id="", es_query={}, callback_url="wss://localhost"))
+        try:
+            environ_backup = os.environ
```

Comment:
technically, you should have this statement *before* the try, but if this statement causes an exception, i guess you're screwed anyway.